### PR TITLE
feat: adds visits find and find_one

### LINKED ISSUE #75: Add pagination support for keyset (cursor) types
The existing Paginator implementation only supports offset pagination. Implement a second class that supports cursors. 

See https://docs.posit.co/connect/cookbook/pagination/ for reference.

### PR DESCRIPTION
**Pull request by tdstein**
Resolves #75
Related to #137

### CODE REVIEW

File: src/posit/connect/cursors.py
Comment (tdstein) [inline src/posit/connect/cursors.py:1]: This is a modified version of the implementation in `paginator.py`. A classic abstraction is possible, but it isn't necessary at this time.

If it's confusing having two separate modules with more-or-less the same exported classes, we could reorganize `curosrs.py` and `paginator.py` into a sub-module.

File: src/posit/connect/visits.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            )
+            for result in results
+        )
+        return next(visits, None)
+
+
+def rename_params(params: dict) -> dict:
+    """Rename params from the internal to the external signature.
+
+    The API accepts `from` as a querystring parameter. Since `from` is a reserved word in Python, the SDK uses the name `start` instead. The querystring parameter `to` takes the same form for consistency.
```
Comment (tdstein) [inline src/posit/connect/visits.py:222]: Note this change! Open to other ideas if there are strong opinions.

File: tests/posit/connect/__api__/v1/instrumentation/content/visits.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (nealrichardson) [inline tests/posit/connect/__api__/v1/instrumentation/content/visits.json:1]: We should have more than one page of mocks to test the feature, right?

File: src/posit/connect/cursors.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+_MAX_PAGE_SIZE = 500
+
+
+@dataclass
+class Page:
+    paging: dict
+    results: List[dict]
+
+
+class Paginator:
```
Comment (nealrichardson) [inline src/posit/connect/cursors.py:18]: Is it confusing that we have two classes called Paginator, just in different submodules? Would it make sense to name these differently?

File: src/posit/connect/cursors.py
Comment (nealrichardson) [reply src/posit/connect/cursors.py:1]: I don't think we need to make them share code or inherit from a common thing. There are only 2.

**Pull request merged**